src/auth_db/mod.rs

```diff
@@ -104,36 +177,48 @@ impl Display for DbError {
 
 impl From<UrlError> for DbError {
     fn from(error: UrlError) -> DbError {
-        DbError::new(format!("URL error: {}", error.description()))
+        DbError::new(format!("URL error: {:?}", error))
     }
 }
 
 impl From<RequestError> for DbError {
     fn from(error: RequestError) -> DbError {
-        DbError::new(format!("request error: {}", error.description()))
+        DbError::new(format!("request error: {:?}", error))
```

Comment:
For a long time I was debugging errors without realising I could do this to get a half-decent error message out.